javaagent/build.gradle.kts

```diff
@@ -218,6 +218,12 @@ tasks {
   named("generateLicenseReport").configure {
     dependsOn(cleanLicenses)
   }
+
+  // We don't publish this jar file, so the actual classifier value is not important.
+  // But we should not have two tasks (jar and shadow) that produce the same file.
+  jar {
+    archiveClassifier.set("slim")
```

Comment:
Taking suggestions for the classifier value :)

Reply:
`empty-do-not-use`? 😄

Reply:
Considering https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5251 slim doesn't look like the best name for now. If this is an intermediate jar that isn't really functional then maybe `-tmp` or something else that hints that this is not supposed to be used.

Reply:
Now only to understand, why smoke test crashes...